find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
```

Comment:
Would it be more readable to rename variable max_index into length?
```suggestion
def find_max(array, length)
```